docs/guide/misc/release-notes.md

```diff
@@ -14,43 +14,103 @@ title: Release Notes
+
+
+Other post-0.7.0-M2 highlights include:
+
+1. New policies: `SshConnectionFailure`, which emits an event if it cannot make
+   an SSH connection to a machine, and `ConditionalSuspendPolicy`, which suspends
+   a target policy if it receives a sensor event.
 
-* First-class Chef integration
+1. Brooklyn reports server features in responses to `GET /v1/server/version`.
 
-* New clouds:  GCE, Softlayer
+1. It is much easier for downstream projects to customise the behaviour of
+   `JcloudsLocationSecurityGroupCustomiser`.
 
-* Networking
+1. Brooklyn is compiled with Java 7 and uses jclouds 1.9.0.
 
-* Docker support:  see [clocker.io](http://clocker.io)
+1. Improvements to the existing Nginx, Riak, RabbitMQ and Bind DNS entities and
+   support for Tomcat 8.
 
 
 ### Backwards Compatibility
 
-* Persistence has been radically overhauled. In most cases the state files from previous versions are compatible,
-  but some items have had to change. For most users this should not be an issue as persistence in the previous version
-  was not working well in any case. 
+Changes since 0.7.0-M2:
 
-* If `brooklyn.webconsole.security.https.required=true` is specified with no explicit port, 
-  it now defaults to 8443; previously it would default to 8081 even in the case of `https`.
+1. Passwords generated with the `generate-password` command line tool must be
+   regenerated. The tool now generates exactly `sha256( salt + password )`.
+
+Changes since 0.6.0:
+
+1. Code deprecated in 0.6.0 has been deleted. Many classes and methods are newly deprecated.
+
+1. Persistence has been radically overhauled. In most cases the state files
+   from previous versions are compatible but many items have had to change.
+
+1. Location configuration getter and setter methods are changed to match those
+   of Entities. This is in preparation for having all Locations be Entities.
+
+1. OpenShift integration has moved from core Brooklyn to the downstream project
+   https://github.com/cloudsoft/brooklyn-openshift.
+
+Please refer to the release notes for versions
+[0.7.0-M2](https://brooklyn.incubator.apache.org/v/0.7.0-M2-incubating/misc/release-notes.html)
+and
+[0.7.0-M1](https://brooklyn.incubator.apache.org/v/0.7.0-M1/start/release-notes.html)
+for further compatibility notes.
 
-* The /v1/catalog/create method now returns a map of ID to item map, instead of an item map, 
-  as the call supports multiple items defined in the YAML.
-  
 
 ### Community Activity
 
-Brooklyn has moved into the Apache Software Foundation.
+During development of 0.7.0 Brooklyn moved to the Apache Software Foundation.
+
+Many exciting projects are using Brooklyn. Notably:
+
+* [Clocker](http://clocker.io), which creates and manages Docker cloud
+  infrastructures.
+
+* The Brooklyn Cloud Foundry Bridge, which brings blueprints into the Cloud
+  Foundry marketplace with the [Brooklyn Service
+  Broker](https://github.com/cloudfoundry-community/brooklyn-service-broker)
```

Comment:
This lives here as of today https://github.com/cloudfoundry-incubator/brooklyn-service-broker